truffle/src/main/ruby/core/rbconfig.rb

```diff
@@ -72,18 +72,14 @@ module RbConfig
       'LIBRUBY_A' => '',
       'LIBRUBYARG' => '',
       'prefix' => '',
-      'ruby_install_name' => 'jruby-truffle',
+      'ruby_install_name' => ruby_install_name,
       'RUBY_SO_NAME' => '$(RUBY_BASE_NAME)'
   }
   
   ruby_home = Truffle::Boot.ruby_home
 
   if ruby_home
-    bindir = if ruby_home.end_with?('/mxbuild/ruby-zip-extracted')
-               File.expand_path('../../bin', ruby_home)
-             else
-               "#{ruby_home}/bin"
-             end
+    bindir = "#{ruby_home}/bin"
```

Comment:
I think the relative symlink will still be wrong in that case inside mxbuild.
Either we need to copy `bin` in the mxbuild (might make sense), or we need to find a way back to `repo/bin`, or make `bin/truffleruby` a symlink to `lib/ruby/truffle/bin/truffleruby`.
IMHO we should ship the launcher and have GraalVM launcher set a few things and delegate to that, so the last option sounds best.